src/repository.cc

```diff
@@ -384,9 +386,9 @@ class StatusWorker {
       path_count = js_paths->Length();
       paths = reinterpret_cast<char **>(malloc(path_count * sizeof(char *)));
       for (unsigned i = 0; i < path_count; i++) {
-        auto js_path = Local<String>::Cast(js_paths->Get(i));
-        paths[i] = reinterpret_cast<char *>(malloc(js_path->Utf8Length() + 1));
-        js_path->WriteUtf8(paths[i]);
+        Nan::Utf8String js_path(Nan::Get(js_paths, i).ToLocalChecked());
+        paths[i] = reinterpret_cast<char *>(malloc(js_path.length() + 1));
+        strcpy(paths[i], *js_path);
```

Comment:
I'm not sure if I did this part correctly.
Please check this part before accepting this patch.